dojo/tools/sonarqube_api/importer.py

```diff
@@ -127,6 +135,93 @@ def import_issues(self, test):
 
         return items
 
+    def import_hotspots(self, test):
+
+        items = list()
+
+        try:
+            product = test.engagement.product
+            config = product.sonarqube_product_set.all().first()
+
+            client = SonarQubeAPI(
+                tool_config=config.sonarqube_tool_config if config else None
+            )
+
+            if config and config.sonarqube_project_key:
+                project = client.get_project(config.sonarqube_project_key)
+            else:
+                project = client.find_project(product.name)
+
+            hotspots = client.find_hotspots(project['key'])
+            logging.info('Found {} hotspots for project {}'.format(len(hotspots), project["key"]))
+
+            for hotspot in hotspots:
+                status = hotspot['status']
+
+                if self.is_reviewed(status):
+                    continue
+
+                type = 'SECURITY_HOTSPOT'
+                if len(hotspot['message']) > 511:
+                    title = hotspot['message'][0:507] + "..."
+                else:
+                    title = hotspot['message']
+                component_key = hotspot['component']
+                line = hotspot.get('line')
+                rule_id = hotspot['key']
+                rule = client.get_hotspot_rule(rule_id)
+                severity = self.convert_sonar_review_priority(rule['vulnerabilityProbability'])
+                description = self.clean_rule_description_html(rule['vulnerabilityDescription'])
+                cwe = self.clean_cwe(rule['riskDescription'])
+                references = self.get_references(rule['riskDescription'])
+
+                sonarqube_issue, _ = Sonarqube_Issue.objects.update_or_create(
+                    key=hotspot['key'],
+                    defaults={
+                        'status': status,
+                        'type': type
+                    }
+                )
+
+                # Only assign the SonarQube_issue to the first finding related to the issue
+                if Finding.objects.filter(sonarqube_issue=sonarqube_issue).exists():
+                    sonarqube_issue = None
+
+                find = Finding(
+                    title=title,
+                    cwe=cwe,
+                    description=description,
+                    test=test,
+                    severity=severity,
+                    references=references,
+                    file_path=component_key,
+                    line=line,
+                    active=True,
+                    verified=self.is_confirmed(status),
+                    false_p=False,
+                    duplicate=False,
+                    out_of_scope=False,
+                    mitigated=None,
+                    mitigation='No mitigation provided',
+                    impact="No impact provided",
+                    numerical_severity=Finding.get_numerical_severity(severity),
```

Comment:
You don't need to set every attribute.
```suggestion
```
`mitigation`, `impact` are not mandatory.
Also `numerical_severity` is calculated in upper layers.